templates/contact-us.html

```diff
@@ -21,23 +21,114 @@
+          open source infrastructure, applications, development and operations. We are proud to deliver the world’s best
+          pathway to open source success.</p>
       </div>
       <div class="col-6 col-start-large-7">
         <p class="p-heading--three">Technical assistance</p>
-        <p class="u-sv1">Our <a href="https://www.ubuntu.com/support">support team</a> team offers a range of services from enterprise support to fully managed infrastructure and applications.</p>
+        <p class="u-sv1">Our <a href="https://www.ubuntu.com/support">support team</a> offers a range of services from enterprise support to fully managed infrastructure and applications.</p>
         <p class="p-heading--three">Trademark and other licenses</p>
-        <p class="u-sv1">Publishing or distributing Ubuntu in any form other than our standard install media and packages requires permission. To license the Ubuntu trademark or any other rights, please <a href="https://www.ubuntu.com/legal/terms-and-policies/contact-us">submit an enquiry</a>.</p>
+        <p class="u-sv1">Publishing or distributing Ubuntu in any form other than our standard install media and packages
+          requires permission. To license the Ubuntu trademark or any other rights, please <a
+            href="https://www.ubuntu.com/legal/terms-and-policies/contact-us">submit an enquiry</a>.</p>
         <p class="p-heading--three">Press and analyst enquiries</p>
         <p class="u-sv1">Please contact <a href="mailto:pr@canonical.com">pr@canonical.com</a>.</p>
         <p class="p-heading--three">Data Privacy enquiries</p>
-        <p class="u-sv1">For information about data privacy, please read our <a href="https://www.ubuntu.com/legal/data-privacy">Privacy Policy</a> or <a href="https://ubuntu.com/legal/data-privacy/enquiry">submit an enquiry</a>. A member of our data privacy team will be in touch with you shortly.</p>
+        <p class="u-sv1">For information about data privacy, please read our <a
+            href="https://www.ubuntu.com/legal/data-privacy">Privacy Policy</a> or <a
+            href="https://ubuntu.com/legal/data-privacy/enquiry">submit an enquiry</a>. A member of our data privacy team
+          will be in touch with you shortly.</p>
         <p class="p-heading--three">Legal enquiries</p>
-        <p>Please note that we cannot provide you with legal advice. For any other queries, please contact us on <a href="mailto:legal@canonical.com">legal@canonical.com</a>.</p>
+        <p>Please note that we cannot provide you with legal advice. For any other queries, please contact us on <a
+            href="mailto:legal@canonical.com">legal@canonical.com</a>.</p>
       </div>
     </div>
   </section>
-{% endblock %}
+  <section class="p-strip--light is-shallow" style="padding-bottom: 6rem;">
+    <div class="u-fixed-width">
+      <p class="p-heading--three">
```

Comment:
```suggestion
      <h2 class="p-heading--three">
```
(and update closing tag on line 56)